docs/users.md

```diff
@@ -522,6 +530,63 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to integrate
+a kind cluster named `ext1` into the same Docker network as your Kubeflex hosting cluster. This setup 
+requires that both clusters are accessible from each other within the shared Docker network.
+
+### Prerequisites
+Ensure that both your hosting and adopted clusters are on the same Docker network. For 
+clusters using the default `kind` docker network, execute the following command to 
+verify connectivity:
+
+```shell
+docker network inspect kind | jq '.[].Containers | to_entries[] | .value.Name'
+```
+
+The output will list the container names for your clusters. For example:
```

Comment:
"For example:" makes me expect that what's next is an example of the output from the above command. But that is not actually what comes next.